### ISSUE TITLE
Comparing auxiliary variables between one latent class and the sample mean?

### ISSUE THREAD

**Issue by DanBennettDev** [cjvanlissa/tidySEM#102]
We're considering an exploratory analysis where we expect to see a class with a particular pattern. We expect this class to have higher than average values on a couple of outcome variables. 
We could use BCH to make pairwise comparisons to each of the other profiles, but this isn't quite the same thing. 
Is there a way of making this comparison in tidySEM?

**Comment by cjvanlissa** [comment]
Yes, you'd need to do BCH but constrain the parameters in all those other classes. That's not a standard option, but you could do it manually.
